akka-http-core/src/test/scala/akka/http/scaladsl/GracefulTerminationSpec.scala

```diff
@@ -199,7 +198,8 @@ class GracefulTerminationSpec
         super.basePoolSettings
           .withTransport(new ClientTransport {
             override def connectTo(host: String, port: Int, settings: ClientConnectionSettings)(implicit system: ActorSystem): Flow[ByteString, ByteString, Future[Http.OutgoingConnection]] = {
-              ClientTransport.TCP.connectTo(host, port, settings)
+              ExampleHttpContexts.proxyTransport(serverBinding.localAddress)
```

Comment:
Does that change anything here? You could probably also just keep using `TCP.connectTo` and pass in the `serverBinding`'s host and port.

Reply:
Good one, indeed